apps/backoffice-v2/src/common/hooks/useCurrentFilter/useCurrentFilter.ts

```diff
@@ -0,0 +1,16 @@
+import { useFilterId } from '@/common/hooks/useFilterId/useFilterId';
+import { useFiltersQuery } from '@/domains/filters/hooks/queries/useFiltersQuery/useFiltersQuery';
+import { useMemo } from 'react';
+
+export const useCurrentFilter = () => {
```

Comment:
A lot of cases where "current" is used the current is implied. What do you think about "selected" or an indication its related to the URL?
```suggestion
export const useSelectedFilter = () => {
```

Reply:
In current context im assuming that some filter is exists and i can retrieve it. I dont care what is source of this filter, if its from URL or selected somewhere by user. So in my opinion `useCurrent` fits well.